File: day6/Python/reverse_words.py
'''
@author: aaditkamat
@date: 27/12/2018
'''

def reverse_words(string):
    new_string = ''
    for word in string.split(' '):
        new_string += word[::-1] + ' '
    return new_string

print("Enter a string: ", end= '')
string = input()
print(reverse_words(string))

File: day6/Ruby/reverse_words.rb
=begin
@author: aaditkamat
@date: 27/12/2018
=end

def reverse_words(string)
    new_string = ''
    string.split(' ').each do |word|
        new_string += word.reverse + ' '
    end
    new_string
end

print"Enter a string: "
string = gets().chomp
print"Reverse of string #{string}:  #{reverse_words(string)}"
